app/dose-notifications/dose-notifications.ts

```diff
@@ -33,8 +33,12 @@ export class DoseNotificationService {
         );
     }
 
-    public cancelAll() {
-        return LocalNotifications.cancelAll();
```

Comment:
This removed notifications that just happened from the phone's notification panel.

Now we only cancel ones that haven't triggered yet (future scheduled ones).